editors/emacs/Portfile

```diff
@@ -272,8 +243,8 @@ if {$subport eq "emacs-app" || $subport eq "emacs-app-devel"} {
             ${destroot}${applications_dir}
         # fix read-permission to resources
         system "chmod a+r ${destroot}${applications_dir}/Emacs.app/Contents/Resources/*"
-        file copy ${filespath}/site-start.el \
-            ${destroot}${applications_dir}/Emacs.app/Contents/Resources/site-lisp
+        file copy ${filespath}/site-start-app.el \
```

Comment:
I guess you shall revbump at least emacs-app-devel after that.

Reply:
No, because the actual installed content doesn't change.